check-environment.js

```diff
@@ -1,22 +1,24 @@
-
-const auth = require('@adobe/jwt-auth')
-const fs = require('fs')
 const fetch = require('node-fetch')
 
 require('dotenv').config()
 
 async function getAccessToken () {
-  const config = {
-    clientId: process.env.CLIENT_ID,
-    technicalAccountId: process.env.TECHNICAL_ACCOUNT_ID,
-    orgId: process.env.ORGANIZATION_ID,
-    clientSecret: process.env.CLIENT_SECRET,
-    metaScopes: [ 'ent_cloudmgr_sdk' ]
+  const form = new FormData();
+  form.append('client_id', process.env.CLIENT_ID);
+  form.append('client_secret', process.env.CLIENT_SECRET);
+  form.append('grant_type', process.env.GRANT_TYPE);
+  form.append('scope', process.env.SCOPES);
+
+  const response = await fetch('https://ims-na1.adobelogin.com/ims/token/v3', {
```

Comment:
I wouldn't recommend doing it this way. Earlier also there were 2 API calls but they were abstract under @adobe/jwt-auth library. I believe there would be a similar adobe library for oauth, please use that

Reply:
[https://github.com/adobe/aio-lib-ims-oauth](https://github.com/adobe/aio-lib-ims-oauth)